### ISSUE TITLE
Target missmatch when using "WASM_PACK_PROFILE"

### ISSUE THREAD

**Issue by extraymond** [catsigma/parcel-plugin-wasm.rs#24]
**Summary**:  not looking for target/wasm/debug when build with dev profile.
**Version**:  1.2.13

**Reproduce steps**:
1. Build with dev profile, "WASM_PACK_PROFILE=dev parcel index.html"
2. wasm-pack successfully build with dev -> produced target/wasm/debug
2. Receive the error not able to find target/wasm/release:

```bash
🚨  project/crate/Cargo.toml: ENOENT: no such file or directory, open 'project/crate/target/wasm32-unknown-unknown/release/rust_parcel.d'
Error: ENOENT: no such file or directory, open 'project/crate/target/wasm32-unknown-unknown/release/rust_parcel.d'

```

Sorry to bother you again. It seems that although the plugin trigger wasm-pack to build with the correct profile, it still watches for compilation target with release profile.

**Comment by extraymond** [comment]
After some investigation, I found that if build with wasm-pack release first, dev mode will work after hot-reloading.

However, if one try to build with dev profile without a release built, it will trigger the above error.

**Comment by catsigma** [comment]
Thanks for this report.

**Comment by catsigma** [comment]
Should be fixed in version `1.2.14`

**Comment by extraymond** [comment]
Thx, it worked without existing release build now.
